File: 1134.cpp
#include <iostream>

using namespace std;

int main(){
    int cod, alcool=0, gasolina=0, diesel=0;
    while (cod != 4){
        cin >> cod;
        switch (cod){
        case 1:
            alcool += 1;
            break;
        case 2:
            gasolina += 1;
            break;
        case 3:
            diesel += 1;
            break;
        default:
            continue;
        }

    }
    cout << "MUITO OBRIGADO" << endl << "Alcool: " << alcool << endl << "Gasolina: " << gasolina << endl << "Diesel: " << diesel << endl;
    return 0;
}

File: 1134.py
alcool = 0
gasolina = 0
diesel = 0
while True:
    combustivel = int(input())
    if combustivel != 4:
        if combustivel == 1:
            alcool += 1
        elif combustivel == 2:
            gasolina += 1
        elif combustivel == 3:
            diesel += 1
    else:
        break
print('MUITO OBRIGADO')
print(f'Alcool: {alcool}')
print(f'Gasolina: {gasolina}')
print(f'Diesel: {diesel}')